controllers/datamovement_controller.go

```diff
@@ -249,8 +291,11 @@ func (r *DataMovementReconciler) Reconcile(ctx context.Context, req ctrl.Request
 	dm.Status.State = nnfv1alpha1.DataMovementConditionTypeRunning
 	cmdStatus := nnfv1alpha1.NnfDataMovementCommandStatus{}
 	cmdStatus.Command = cmd.String()
+	cmdStatus.MPIHostfilePath = mpiHostfile
+	cmdStatus.MPIHostfileContents = mpiHostfileContents
 	dm.Status.CommandStatus = &cmdStatus
 	log.Info("Running Command", "cmd", cmdStatus.Command)
+	log.Info("Using hostfile", "file path", mpiHostfile, "contents", cmdStatus.MPIHostfileContents)
```

Comment:
Remember when you are running on a single rabbit that "contents" might be one line, but this could be hundreds of entries. 

Logs aren't infinite in size.

Reply:
I've removed this.